invenio_curations/notifications/generators.py

```diff
@@ -19,26 +23,41 @@
 class GroupMembersRecipient(RecipientGenerator):
     """Group/Role member recipient generator for notifications."""
 
-    def __init__(self, key):
-        """Ctor."""
+    def __init__(self, key: str) -> None:
+        """Initialize the generator with a key.
+
+        Args:
+            key: The context key to look up the group/role information
+        """
         self.key = key
 
-    def __call__(self, notification, recipients: dict):
-        """Fetch group members as recipients."""
-        group = dict_lookup(notification.context, self.key)
+    def __call__(
+        self, notification: Notification, recipients: dict[str, Recipient]
```

Comment:
should we follow https://docs.astral.sh/ruff/rules/missing-trailing-comma/ ?
black would reformat the parameter in a way that they are below each other.

Reply:
i used` black .` seems was not applied! Ill check